src/components/TaxPayer/TaxPayer.tsx

```diff
@@ -95,57 +95,58 @@ export default function TaxPayerInfo ({ navButtons, onAdvance }: PagedFormProps)
         />
       </div>
     )
-  }
+  })()
 
   return (
-    <Box display="flex" justifyContent="center">
-      <form onSubmit={handleSubmit(onSubmit)}>
-        <Box display="flex" justifyContent="flex-start">
-          <h2>Taxpayer Information</h2>
-        </Box>
-
-        <h4>Primary Taxpayer Information</h4>
-        <PersonFields
-          register={register}
-          errors={errors}
-          defaults={taxPayer?.primaryPerson}
-        />
-        <LabeledInput
-          label="Address"
-          name="address.address"
-          register={register}
-          required={true}
-          error={errors.address?.address}
-          defaultValue={taxPayer?.primaryPerson?.address.address}
-        />
-        <LabeledInput
-          label="Unit No"
-          register={register}
-          name="address.aptNo"
-          required={false}
-          error={errors.address?.aptNo}
-          defaultValue={taxPayer?.primaryPerson?.address.aptNo}
-        />
-        <LabeledInput
-          label="City"
-          register={register}
-          name="address.city"
-          patternConfig={Patterns.name}
-          required={true}
-          error={errors.address?.city}
-          defaultValue={taxPayer?.primaryPerson?.address.city}
-        />
-        <LabeledCheckBox
-          label="Check if you have a foreign address"
-          control={control}
-          value={isForeignCountry}
-          setValue={updateForeignCountry}
-          name="isForeignCountry"
-        />
-        {csz}
+    <PagerContext.Consumer>{({ navButtons, onAdvance }) =>
```

Comment:
Can the pagercontext lines here be separated like below
```
<PagerContext.Consumer>
      {({ onAdvance, navButtons }) =>
          // somecode
      }
</PagerContext.Consumer>
```

Reply:
same in createPDF, F1099Info, W2JobInfo

Reply:
For sure, I will do that. The reason I didn't do it before was to try to minimize the diff. I didn't want to have to indent the entire block. But final code quality is more important. Will do.

Reply:
did that.